app/i18n/locales/en.json

```
@@ -744,5 +744,10 @@
   "Incoming_call_from": "Incoming call from",
   "Call_started": "Call started",
   "Message_has_been_shared":"Message has been shared",
-  "No_channels_in_team": "No Channels on this team"
+  "No_channels_in_team": "No Channels on this team",
+  "Jitsi_may_requires_authentication": "Jitsi may requires authentication",
```

Apply the updated string where used.

```suggestion
  "Jitsi_may_require_authentication": "Jitsi may require authentication",
```